docs/DeveloperGuide.md

```diff
@@ -700,35 +700,110 @@ Given below are instructions to test the app manually.
 
 <div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
 testers are expected to do more *exploratory* testing.
-
 </div>
 
-### Launch and shutdown
-
-1. Initial launch
-
-   1. Download the jar file and copy into an empty folder
-
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+### General
 
-   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+#### Launching TutorBuddy
+1. Download the jar file and copy into an empty folder
+1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
+#### Saving Window Preferences
+1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+1. Re-launch the app by double-clicking the jar file.<br>
+   Expected: The most recent window size and location is retained.
 
-### Deleting a student
+#### Clearing all data
 
-1. Deleting a student while all students are being shown
-
-   1. Prerequisites: List all students using the `list` command. Multiple students in the list.
-
-   1. Test case: `delete_student 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-
-   1. Test case: `delete_student 0`<br>
-      Expected: No student is deleted. Error details shown in the status message. Status bar remains the same.
+<div markdown="span" class="alert alert-info">
+:information_source: **WARNING:** Do not clear all data if you have important information in TutorBuddy!
+</div>
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+1. Prerequisites: None.
+1. Test case: `clear`<br>
+   Expected: Student list view and Session list view becomes empty.
+1. To get back the sample data in TutorBuddy, simply delete the data folder in the home folder and restart TutorBuddy.
+
+#### Listing all students and sessions
+1. Prerequisites: None.
+1. Test case: `list`<br>
+   Expected: List all students and sessions on the Student list view and the Session list view if any.
```

Comment:
May want to indicate that this will be on the tuition tab.

Reply:
Good point!